client/src/component/Dashboard.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import CodeWarsCard from "/CodeWarsCard";
+
+const Dashboard = () => {
+  return (
+    <div className="dashboard">
+      {/* other dashboard components */}
+      <CodeWarsCard rank="6kyu" progress={75} milestone="5kyu" />
+      {/* other dashboard components */}
+    </div>
+  );
+};
+
+export default Dashboard;
```

Comment:
delete line 14 export default Dashboard; because you are using export in line 4 after your chenges.